core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -344,9 +335,8 @@ public synchronized void suspend() throws IOException {
    *
    * @param sequence sequence to catch up
    * @return the catch-up task
-   * @throws IOException
    */
-  public synchronized CatchupFuture catchup(long sequence) throws IOException {
+  public synchronized CatchupFuture catchup(long sequence) {
     Preconditions.checkState(mSuspended, "journal is not suspended");
     Preconditions.checkState(mState.get() == State.STANDBY, "unexpected state " + mState.get());
```

Comment:
can you also fix this to avoid unnecessary string concat in common cases ?
```suggestion
    Preconditions.checkState(mState.get() == State.STANDBY, "unexpected state %s", mState.get());
```